tests/end-to-end/cli/child-process-output.phpt

```diff
@@ -0,0 +1,23 @@
+--TEST--
+ShutdownHandler does not output when child process exits
+--FILE--
```

Comment:
This needs a `--SKIPIF--` section to skip the test when the `pcntl` extension is not available.

Reply:
I've added the `--SKIPIF--`